java/src/test/java/io/ably/lib/test/common/jre/JreResourceLoader.java

```diff
@@ -10,7 +10,7 @@
 	public byte[] read(String resourceName) throws IOException {
 		FileInputStream fis = null;
 		try {
-			fis = new FileInputStream(new File("src/test/resources", resourceName));
+			fis = new FileInputStream(new File("../lib/src/test/resources", resourceName));
```

Comment:
Er, I mean no, that path must be relative to the subproject, even though I can run the tests in the root of the ably-java checkout. It seems to work.